src/core/reducers/api.js

```diff
@@ -79,13 +79,16 @@ export const initialApiState: ApiStateType = {
   userAgentInfo: { browser: {}, os: {} },
 };
 
+type Action =
+  | SetAuthTokenAction
+  | SetLangAction
+  | SetClientAppAction
+  | SetUserAgentAction
+  | LogOutUserAction;
+
 export default function api(
   state: Exact<ApiStateType> = initialApiState,
-  action: SetAuthTokenAction
-    & SetLangAction
```

Comment:
Hmm, I've not seen that before. Why were we using `&` before? Was that just a mistake that Flow didn't previously catch?

Reply:
We use `&` for intersections but the behavior has changed (got stricter). In short, yes it was a mistake not caught by Flow. It did work because there was no strict type on the redux `type` properties I think.